src/Text/Pandoc/Readers/Vimwiki.hs

```diff
@@ -555,17 +547,14 @@ link = try $ do
                   then do
                     url <- manyTillChar anyChar $ char '|'
                     lab <- mconcat <$> manyTill inline (string "]]")
-                    let tit = if isURI url
-                                 then ""
-                                 else "wikilink"
-                    return $ B.link (procLink url) tit lab
+                    return $ B.linkWith (attr url) (procLink url) "" lab
                   else do
                     manyTill anyChar (string "]]")
 -- not using try here because [[hell]o]] is not rendered as a link in vimwiki
-                    let tit = if isURI contents
-                                 then ""
-                                 else "wikilink"
-                    return $ B.link (procLink contents) tit (B.str contents)
+                    return $ B.linkWith (attr contents) (procLink contents) "" (B.str contents)
```

Comment:
Can you comment on the decision to add a substantive title here?

Reply:
I'm afraid you've misread, title is still `""` here.

Reply:
oh, right.